03-single-hop-ipv6-icmp/03-single-hop-ipv6-icmp.md

```diff
@@ -62,3 +62,21 @@ simultaneously to one native node.
 All nodes are still running, reachable, and the packet buffer is empty 10
 seconds after completions (use module `gnrc_pktbuf_cmd`).
 Packet loss is irrelevant in this stress test.
+
+Task #04 - ICMPv6 stress test on native (neighbor cache stress)
```

Comment:
Just realize that this should be task 0**5** (Don't multiple things at the same time)

Reply:
I realized that as well, when I was updating https://github.com/RIOT-OS/Release-Specs/issues/105. Fixed in https://github.com/RIOT-OS/Release-Specs/pull/106

Reply:
I realized that someone already edited #105 when I was doing...